cmd/zenchron-engineering/selfhost.go

```
@@ -0,0 +1,428 @@
+		return "", "", nil, fmt.Errorf("close report schema: %w", err)
+	}
+	cleanup := func() { os.Remove(report.Name()); os.Remove(schema.Name()) }
+	return report.Name(), schema.Name(), cleanup, nil
+}
+
+func writeComment(target issue, branch string, pr pullRequest, head string, report executorReport) (string, error) {
+	file, err := os.CreateTemp("", "zenchron-handoff-*.md")
+	if err != nil {
+		return "", fmt.Errorf("create handoff comment: %w", err)
+	}
+	defer file.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `file.Close` is not checked

(errcheck)